### PR TITLE
Test setup

### PR DESCRIPTION
**Pull request by kernelwhisperer**

### CODE REVIEW

**Comment by kernelwhisperer** [comment]
@jainkunal ready for review

File: package.json [hunk 1]
```diff
@@ -1,87 +1,90 @@
...
+        "ts-node": "^10.7.0",
+        "typechain": "^8.1.0",
+        "typescript": "^4.6.3",
+        "vite": "^3.2.4",
+        "vitest": "^0.24.3",
+        "wido": "^0.0.33",
+        "yarn-upgrade-all": "^0.7.1"
+    },
+    "dependencies": {
+        "@google-cloud/storage": "^6.5.4",
```
Comment (jainkunal) [inline package.json:87]: Is it required?

File: test/utils/test-utils.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import {parseEther} from "@ethersproject/units";
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
+import {BigNumber, Signer} from "ethers";
+import hre from "hardhat";
+import {ethers} from "hardhat";
+import erc20ABI from "../../abi/erc20.json";
+import {IWidoRouter} from "../../typechain";
+
+const whaleAddress: {[key: string]: string} = {
+  // Mainnet
```
Comment (jainkunal) [inline test/utils/test-utils.ts:11]: Can we remove most of it and only keep USDC for mainnet?

**Review by jainkunal — APPROVED** [review]
Couple of comments

File: test/utils/test-utils.ts (hunk 2 above)
Comment (kernelwhisperer) [reply test/utils/test-utils.ts:11]: I kept WETH, which we use, and USDC on a couple of networks.

**Pull request merged**